geode-management/src/main/java/org/apache/geode/management/runtime/RuntimeInfo.java

```diff
@@ -20,5 +20,14 @@
 import com.fasterxml.jackson.annotation.JsonTypeInfo;
 
 @JsonTypeInfo(use = JsonTypeInfo.Id.CLASS, property = "class")
-public interface RuntimeInfo extends Serializable {
+public abstract class RuntimeInfo implements Serializable {
+  private String memberName;
+
+  public void setMemberName(String memberName) {
+    this.memberName = memberName;
+  }
+
+  public String getMemberName() {
+    return memberName;
+  }
```

Comment:
what would be returned from `getMemberName` on a cache element that uses global runtime info?

Reply:
In `CacheRealizationFunction` we have something like this:
```// set the membername if this is not a global runtime
    if (cacheElement instanceof CorrespondWith
        && !((CorrespondWith) cacheElement).isGlobalRuntime()) {
      runtimeInfo.setMemberName(context.getMemberName());
    }```